parser_test.go

```diff
@@ -3142,7 +3142,7 @@ func TestAllowedFunctions(t *testing.T) {
 		tests = append(tests, testCase{
 			name:     allowedFunction,
 			stmt:     fmt.Sprintf("select %s from t", functionCall),
-			deparsed: fmt.Sprintf("select %s from t", functionCall),
+			deparsed: fmt.Sprintf("select %sfrom t", functionCall),
```

Comment:
`functionCall` is always going to end with a parenthesis, hence the deparsed value should not have a space.